src/integrationTest/java/org/opensearch/security/privileges/int_tests/DashboardMultiTenancyIntTests.java

```diff
@@ -0,0 +1,794 @@
+                bulkBody,
+                new BasicHeader("securitytenant", "human_resources")
+            );
+
+            assertThat(
+                response,
+                containsExactly(dashboards_index_human_resources).at("items[*].delete[?(@.result == 'deleted')]._index")
+                    .reducedBy(user.reference(WRITE))
+                    .whenEmpty(isOk())
+            );
+        } finally {
+            delete(
+                dashboards_index_human_resources.name() + "/_doc/" + testDocId1,
+                dashboards_index_human_resources.name() + "/_doc/" + testDocId2
+            );
+        }
+    }
+
+    @Test
+    public void bulkWithUpdate_withTenantHeader_humanResources() {
+        String testDocId1 = "test_bulk_update_doc_1_" + UUID.randomUUID();
+        String testDocId2 = "test_bulk_update_doc_2_" + UUID.randomUUID();
+
+        try (TestRestClient restClient = cluster.getRestClient(user); TestRestClient adminRestClient = cluster.getAdminCertRestClient()) {
+
+            // Create documents first
+            adminRestClient.put(
+                dashboards_index_human_resources.name() + "/_doc/" + testDocId1 + "?pretty",
+                json("foo", "bar1"),
+                new BasicHeader("securitytenant", "human_resources")
+            );
+            adminRestClient.put(
+                dashboards_index_human_resources.name() + "/_doc/" + testDocId2 + "?pretty",
+                json("foo", "bar2"),
+                new BasicHeader("securitytenant", "human_resources")
+            );
+
+            String bulkBody = """
+                { "update" : { "_index" : ".kibana", "_id" : "%s" } }
+                { "doc" : { "foo" : "updated1" } }
+                { "update" : { "_index" : ".kibana", "_id" : "%s" } }
+                { "doc" : { "foo" : "updated2" } }
+                """.formatted(testDocId1, testDocId2);
+
+            TestRestClient.HttpResponse response = restClient.postJson(
+                "_bulk?pretty",
+                bulkBody,
+                new BasicHeader("securitytenant", "human_resources")
+            );
+
+            if (user.reference(WRITE).covers(dashboards_index_human_resources)) {
+                if (user == GLOBAL_TENANT_READ_WRITE_USER || user == WILDCARD_TENANT_USER) {
+                    assertThat(response, isOk());
+                    assertThat(
+                        response,
+                        containsExactly(dashboards_index_human_resources).at("items[*].update[?(@.result == 'updated')]._index")
+                            .reducedBy(user.reference(WRITE))
+                            .whenEmpty(isOk())
+                    );
+                } else {
+                    // At the moment, we get here a nested error:
+                    // Update is not supported when FLS or DLS or Fieldmasking is activated
+                    // This is a bug; even though it does not seem to have an impact on Dashboards functionality
```

Comment:
I've seen this come up on a separate repo here: https://github.com/opensearch-project/observability/issues/1951

Could this be related?